server/src/common.ts

```diff
@@ -159,6 +161,12 @@ class Config implements IConfig.Main {
                 this.server.port = port;
             }
         }
+        if (process.env.WS_PORT) {
+            const wsPort = parseInt(process.env.WS_PORT!, 10);
```

Comment:
exclamation point?

Reply:
The WS_PORT env var turned out to be unnecessary.  It's gone now.